### PR TITLE
Data disk doc update

### PR DESCRIPTION
**Pull request by schaefi**
Update data-disk manual page
    
The man page for this command was not in line with the current state of the command and its subcommands.

### CODE REVIEW

**Comment by schaefi** [comment]
@bear454 Thanks for the offer to take over this one, but I felt I messed it up, this I should also fix it ;)
Would be great if you can have a look at it. Thanks

**Comment by schaefi** [comment]
review ?

File: doc/man/azurectl::compute::data_disk.md [hunk 1]
```diff
@@ -7,17 +7,21 @@ hard disk (VHD) image in Azure storage.
 
 # SYNOPSIS
 
-__azurectl__ compute data-disk create --cloud-service-name=*name* --size=*disk-size-in-GB*
+__azurectl__ compute data-disk create --disk-basename=*name*
+
+    [--size=disk-size-in-GB]
+    [--label=label]
+
+__azurectl__ compute data-disk delete --disk-name=*name*
```
Comment (schaefi) [inline doc/man/azurectl::compute::data_disk.md]: changed ??

File: doc/man/azurectl::compute::data_disk.md [hunk 2]
```diff
@@ -7,17 +7,21 @@ hard disk (VHD) image in Azure storage.
...
+
+__azurectl__ compute data-disk attach --cloud-service-name=*name* --disk-name=*name*
 
     [--instance-name=name]
     [--label=label]
```
Comment (schaefi) [inline doc/man/azurectl::compute::data_disk.md]: - on create the label is set into the metadata of the created vhd disk
- on attach the label would be set into the metadata of the created data-disk

we might consider to change the option names into --disk-label and --data-disk-label

Thoughts ?

**Comment by schaefi** [comment]
:smile: thanks for the merge

**Comment by bear454** [comment]
Thanks for fixing it. Will you `bumbversion patch` and do a release, or wait for whatever's next?

**Pull request merged**